### PR TITLE
Netsuite landing page

### PR DESCRIPTION
**Pull request by NileshPant1999**

### CODE REVIEW

File: src/app/core/models/enum/enum.model.ts [hunk 1]
```diff
@@ -74,7 +74,8 @@ export enum AppName {
   INTACCT = 'Sage Intacct',
   SAGE300 = 'Sage 300 CRE',
   QBO = 'QuickBooks Online',
-  BUSINESS_CENTRAL = 'Dynamics 365 Business Central'
+  BUSINESS_CENTRAL = 'Dynamics 365 Business Central',
+  NETSUITE = 'Netsuite'
```
Comment (ashwin1111) [inline src/app/core/models/enum/enum.model.ts:78]: NetSuite

File: src/app/integrations/netsuite/netsuite-onboarding/netsuite-onboarding-landing/netsuite-onboarding-landing.component.html [hunk 2]
```diff
@@ -1 +1,8 @@
-<p>netsuite-onboarding-landing works!</p>
+<div>
+    <div>
+        <app-landing-page-header [iconPath]="'assets/logos/netsuite-logo.png'" [appName]="'Netsuite'" [appDescription]="'Import data from Netsuite to ' + brandingConfig.brandName + ' and Export expenses from ' + brandingConfig.brandName + ' to Netsuite. '" [isLoading]="false" [isIntegrationSetupInProgress]="false" [isIntegrationConnected]="false" [redirectLink]="redirectLink" [buttonText]="'Connect'" [postConnectionRoute]="'netsuite/onboarding/connector'"></app-landing-page-header>
```
Comment (ashwin1111) [inline src/app/integrations/netsuite/netsuite-onboarding/netsuite-onboarding-landing/netsuite-onboarding-landing.component.html:3]: NetSuite

File: src/app/integrations/netsuite/netsuite-onboarding/netsuite-onboarding-landing/netsuite-onboarding-landing.component.ts [hunk 3]
```diff
@@ -7,6 +9,12 @@ import { Component, OnInit } from '@angular/core';
 })
 export class NetsuiteOnboardingLandingComponent implements OnInit {
 
+  brandingConfig = brandingConfig;
+
+  redirectLink = brandingKbArticles.onboardingArticles.QBO.CONNECTOR;
```
Comment (ashwin1111) [inline src/app/integrations/netsuite/netsuite-onboarding/netsuite-onboarding-landing/netsuite-onboarding-landing.component.ts:14]: use right keys from start, at last we can just change values

File: src/app/integrations/netsuite/netsuite-onboarding/netsuite-onboarding-landing/netsuite-onboarding-landing.component.ts [hunk 4]
```diff
@@ -7,6 +9,12 @@ import { Component, OnInit } from '@angular/core';
...
+
+  embedVideoLink = brandingDemoVideoLinks.onboarding.QBO;
```
Comment (ashwin1111) [inline src/app/integrations/netsuite/netsuite-onboarding/netsuite-onboarding-landing/netsuite-onboarding-landing.component.ts:16]: same as above

File: src/assets/logos/netsuite-logo.png
Comment (ashwin1111) [inline src/assets/logos/netsuite-logo.png:1]: this asset might be used for existing view, pls don't delete them

File: src/app/integrations/netsuite/netsuite-onboarding/netsuite-onboarding-landing/netsuite-onboarding-landing.component.ts (hunk 4 above)
Comment (NileshPant1999) [reply src/app/integrations/netsuite/netsuite-onboarding/netsuite-onboarding-landing/netsuite-onboarding-landing.component.ts:16]: we don't have a video @ashwin1111

Comment (NileshPant1999) [reply src/app/integrations/netsuite/netsuite-onboarding/netsuite-onboarding-landing/netsuite-onboarding-landing.component.ts:16]: once i get the video i will update

Comment (ashwin1111) [reply src/app/integrations/netsuite/netsuite-onboarding/netsuite-onboarding-landing/netsuite-onboarding-landing.component.ts]: don't use QBO key, create new key for netsuite and use same link as QBO

**Pull request merged**